src/components/shared/ExampleAccordion.vue

```diff
@@ -0,0 +1,277 @@
+  },
+  data() {
+    return {
+      open: false,
+      deleteDialog: null,
+    };
+  },
+  computed: {
+    ...mapState({
+      repository: state => state.Repository.selectedRepository,
+    }),
+    entitiesList() {
+      return getEntitiesList(this.entities)
+        .map(entity => ({
+          value: entity,
+          class: this.getEntityClass(entity),
+          label: this.getEntityLabel(entity),
+        }));
+    },
+  },
+  methods: {
+    ...mapActions([
+      'deleteEvaluateExample',
+    ]),
+    getEntityClass(entity) {
+      const color = getEntityColor(
+        entity,
+        this.repository.entities || this.repository.entities_list,
+        this.entities,
+      );
+      return `entity-${color}`;
+    },
+    getEntityLabel(entityName) {
+      return this.entities.reduce((current, e) => {
+        if (e.entity === entityName) {
+          return e.label;
+        }
+        return current;
+      }, 'unlabeled');
+    },
+    deleteThisExample() {
+      return new Promise((resolve, reject) => {
+        this.deleteDialog = this.$dialog.confirm({
+          message: 'Are you sure? The example will be deleted.',
+          confirmText: 'Delete',
+          type: 'is-danger',
+          onConfirm: async () => {
+            await this.deleteEvaluateExample({ id: this.id });
+            this.$emit('deleted');
+            resolve();
+          },
+          onCancel: () => {
+            /* istanbul ignore next */
+            reject();
+          },
+        });
+      });
+    },
+  },
+};
+</script>
+
+
+<style lang="scss" scoped>
+  @import '../../assets/scss/utilities.scss';
+
+  .beforeBorder {
+    position: relative;
+
+    &:before {
+      position: absolute;
+      bottom: -1px;
+      left: 0;
+      width: 100%;
+      content: '';
+      border-bottom: 1px solid #2bbfac;
+      transition: opacity .1s linear, transform .5s ease-in-out;
+    }
+
+    &:not(:hover)::before {
+      opacity: 0;
+      transform: scaleX(0);
+    }
+  }
+
+  .Accordion {
+    overflow: hidden;
+    background: #fff;
+    box-shadow: 0 1px 12px 1PX rgba(0,0,0,0.25);
+  }
+
+  .expander {
+    &__trigger {
+      display: flex;
+      justify-content: space-between;
+      padding: .7rem;
+      margin: .5rem .8rem 0;
+      cursor: pointer;
+      border: 1px solid #cfd5d9;
+      border-radius: 3px;
+
+      &__btns-wrapper {
+        display: flex;
+        justify-content: space-between;
+      }
+    }
+
+    &__body {
+      padding: .7rem 0;
+      margin: 0 .8rem;
+      background: #f5f5f5;
+    }
+  }
+
+  .fade-enter-active, .fade-leave-active {
+    transition: opacity .2s;
+  }
+
+  .example {
+    $radius: 8px;
+
+    margin: 16px 8px;
+    overflow: visible;
+    background-color: $white-bis;
+    border-radius: $radius;
+
+    &__icon {
+      margin: 0 .5rem;
+
+      &:hover {
+        color: black;
+        transition: 1s;
+      }
+    }
+
+    &-text {
+      display: flex;
+      padding: 8px 16px;
```

Comment:
Try to use rem instead of px. actually, 1 rem have 16px; maybe this link can be helpful: http://www.standardista.com/px-to-rem-conversion-if-root-font-size-is-16px/